### DISCUSSION TITLE
Control bots multiplayer setting

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12445] [Ideas and suggestions]
Basically let us optionally control bots in multiplayer games. It could have restrictions like karma or only controlling assistants as options.

**Comment by erunehtar** [comment]
Hi, I implemented the feature directly in the game source code rather than a mod. Changes can be found in the branch here: https://github.com/erunehtar/Barotrauma/tree/allow-controlling-bots-in-multiplayer if you want to try it.

**Reply by erunehtar** [reply]
pull request: https://github.com/FakeFishGames/Barotrauma/pull/17843
